package.json

```diff
@@ -4,28 +4,29 @@
   "version": "0.0.14",
   "description": "truffle - Simple development framework for Ethereum",
   "dependencies": {
-    "coffee-script": "*",
```

Comment:
All I did was npm install colors --save.  Thanks npm. 

PS. Don't merge this quite yet, updated the description to indicate its a WIP.